include/gsl/multi_span

```diff
@@ -109,107 +109,127 @@ public:
     using reference = std::add_lvalue_reference_t<value_type>;
     using const_reference = std::add_lvalue_reference_t<std::add_const_t<value_type>>;
 
-    constexpr multi_span_index() GSL_NOEXCEPT {}
+    constexpr multi_span_index() noexcept {}
+
 
-    constexpr multi_span_index(const value_type (&values)[Rank]) GSL_NOEXCEPT
+    constexpr multi_span_index(const value_type (&values)[Rank]) noexcept
     {
+        GSL_SUPPRESS(bounds.1) // NO-FORMAT: attribute
+        GSL_SUPPRESS(bounds.3) // NO-FORMAT: attribute
         std::copy(values, values + Rank, elems);
     }
 
     template <typename... Ts, typename = std::enable_if_t<(sizeof...(Ts) == Rank) &&
                                                           details::are_integral<Ts...>::value>>
-    constexpr multi_span_index(Ts... ds) GSL_NOEXCEPT : elems{narrow_cast<value_type>(ds)...}
-    {
-    }
+    constexpr multi_span_index(Ts... ds) noexcept : elems{narrow_cast<value_type>(ds)...}
+    {}
 
-    constexpr multi_span_index(const multi_span_index& other) GSL_NOEXCEPT = default;
+    constexpr multi_span_index(const multi_span_index& other) noexcept = default;
 
-    constexpr multi_span_index& operator=(const multi_span_index& rhs) GSL_NOEXCEPT = default;
+    constexpr multi_span_index& operator=(const multi_span_index& rhs) noexcept = default;
 
     // Preconditions: component_idx < rank
+    GSL_SUPPRESS(bounds.2) // NO-FORMAT: attribute
+    GSL_SUPPRESS(bounds.4) // NO-FORMAT: attribute
     constexpr reference operator[](std::size_t component_idx)
     {
         Expects(component_idx < Rank); // Component index must be less than rank
         return elems[component_idx];
     }
 
     // Preconditions: component_idx < rank
-    constexpr const_reference operator[](std::size_t component_idx) const GSL_NOEXCEPT
+    GSL_SUPPRESS(bounds.2) // NO-FORMAT: attribute
+    GSL_SUPPRESS(bounds.4) // NO-FORMAT: attribute
+    constexpr const_reference operator[](std::size_t component_idx) const noexcept
     {
         Expects(component_idx < Rank); // Component index must be less than rank
         return elems[component_idx];
     }
 
-    constexpr bool operator==(const multi_span_index& rhs) const GSL_NOEXCEPT
+    GSL_SUPPRESS(bounds .1) // NO-FORMAT: attribute
```

Comment:
Here and in the next line we could remove the space in "bounds ." and align it with the format in the other suppresses.

Reply:
There seems to be an issue with clang-format where it inserts the space even when it is supposed to ignore the commented line. Will fix.